cmd/monaco/test-resources/integration-all-configs/project/calculated-metrics-application-mobile/metrics-mobile-app.yaml

```diff
@@ -6,6 +6,4 @@ speed:
   - appId: "coming from override"
 
 speed.environment1:
-  # temporarily created by hand in the integration test environment
-  # will be fixed in https://github.com/dynatrace-oss/dynatrace-monitoring-as-code/issues/275
-  - appId: "MOBILE_APPLICATION-23395670912B4E78"
+  - appId: "/only-post/application-mobile/mobile-application.id"
```

Comment:
Since you also fixed #275 alongside this, you can get rid of the `only-post` project. This has the additional benefit that the update of mobile app configs would also be tested in the integration tests.